app/src/main/java/com/example/tasky/agenda_details/domain/ImageCompression.kt

```diff
@@ -0,0 +1,23 @@
+package com.example.tasky.agenda_details.domain
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.graphics.drawable.BitmapDrawable
+import android.graphics.drawable.Drawable
+import android.net.Uri
```

Comment:
Having lots of Android imports here in domain which is not allowed since it should be isolated 😄 
I'd recommend to leave this up to the data layer. You can work with an abstraction that you put in domain, like
```
interface ImageCompressor {
    suspend fun compress(bytes: ByteArray): ByteArray
}
```